test/new-e2e/tests/otel/otlp-ingest/operation_and_resource_name_logic_v2_test.go

```diff
@@ -0,0 +1,170 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package otlpingest
+
+import (
+	awskubernetes "github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments/aws/kubernetes"
+	"testing"
+
+	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/environments"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/otel/utils"
+)
+
+type opNameV2RecvrV1TestSuite struct {
+	e2e.BaseSuite[environments.Kubernetes]
+}
+
+func TestOpAndResNameV2WithSpanRecvrV1(t *testing.T) {
```

Comment:
nit: Would prefer for all of the tests in this file to be prefixed with `TestOTLPIngest` so that they can be sorted in the gitlab job output